desktop/apps/collect/client.coffee

```diff
@@ -61,6 +62,12 @@ module.exports.init = ->
     categoryMap: sd.CATEGORIES
 
   # Main Artworks view
+  if @user?.hasLabFeature('Keyword Search')
```

Comment:
Hmm, how does this get initialized?

You could look at `sd.CURRENT_USER?` for that, but that's just going to be an object not a model.

You could require the current user model, and then do:

```
user = CurrentUser.orNull()
if user?.hasLabFeature...
```

Check out that [orNull](https://github.com/artsy/force/blob/d6b4e7660033449bf2f2755b486c1b9cb3fe3ac2/lib/current_user.coffee#L282) helper.